tokio/src/util/ready.rs

```diff
@@ -1,3 +1,4 @@
+#[allow(unused_macros)]
```

Comment:
Yeah, `cargo hack --each-feature` fails otherwise on the `--no-default-features` `integration-test` step. I have no idea what causes it though and I don’t know how to reproduce it with just Cargo commands.